CHANGES.md

```diff
@@ -97,6 +97,7 @@
 * Added support for avro payload format in Beam SQL Pubsub Table ([BEAM-5504](https://issues.apache.org/jira/browse/BEAM-5504))
 * Added option to disable unnecessary copying between operators in Flink Runner (Java) ([BEAM-11146](https://issues.apache.org/jira/browse/BEAM-11146))
 * Added CombineFn.setup and CombineFn.teardown to Python SDK. These methods let you initialize the CombineFn's state before any of the other methods of the CombineFn is executed and clean that state up later on. If you are using Dataflow, you need to enable Dataflow Runner V2 by passing `--experiments=use_runner_v2` before using this feature. ([BEAM-3736](https://issues.apache.org/jira/browse/BEAM-3736))
+* Added Docker based developer build environment. ([BEAM-10891](https://issues.apache.org/jira/browse/BEAM-10891))
```

Comment:
I don't think this belongs in CHANGES.md, it is intended to communicate changes in the Beam release to users, but this change is just for developers.

Reply:
Ok, I'll remove it.